lineage/figures/figureCommon.py

```diff
@@ -34,6 +36,13 @@
 num_data_points = 50
 
 
+def lineage_good_to_analyze(tmp_lineage):
```

Comment:
checker function that checks whether or not a lineage is good for analysing

Reply:
Something like your comment here should be the docstring.